### PR TITLE
Outline for getting started with electron

### PR DESCRIPTION
**Pull request by brianamarie**
## Overview
**TL;DR**
This is the outline for Electron: Building an App From Scratch 

_Note: This will be merged into the `electron-course` branch, the dev branch for the public ship of the electron course._

### Questions
@github/services-training @lee-dohm: 
- Are these the right topics? 
- What questions should we be answering with this unit? 
- What should the breakdown of topics look like? 

### Next Steps
- Get review and 👍 on topics and breakdown of topics 

### Review
@github/services-training and @lee-dohm for review, comments, and tweaks on specific questions above. Please feel free to mention or bring in anyone else who may have good feedback here.

### CODE REVIEW

**Comment by brianamarie** [comment]
To try to honor the deadlines set for the outlines, and so that @github/services-training can start working on the copies, I'm going to merge this into the `electron-course` branch.
